bundles/media/imagine-adapter.rst

```diff
@@ -0,0 +1,77 @@
+Imagine adapter
+===============
+
+The rendering engine for media is able to use different imagine adapters. By default the bundle detects the installed
+adapters and uses the most mature implementation by default.
+
+Available adapters are:
+
+* `gd`_
+* `imagick extension`_
+* `vips extension`_
+* `contao/imagine-svg`_
+
+GD
+--
+
+The simplest adapter is GD and it has no external dependencies besides the PHP extension.
+
+Imagick
+-------
+
+This extension uses imagemagick to render the images.
+
+To install it use following commands:
+
+__Mac:__
+
+.. code-block:: bash
+
+    brew install imagemagick
+    pecl install imagick
+
+__Linux:__
+
+.. code-block:: bash
+
+    apt-get install libmagickwand-dev inkscape
+    pecl install imagick
+
+VIPS
+----
+
+This extension uses vips to render the images.
+
+To install it use following commands:
+
+__Mac:__
+
+.. code-block:: bash
+
+    brew install imagemagick
+    pecl install imagick
```

Comment:
Are you sure the above two lines are correct?

Reply:
yes its correct :)